docs/guides/security/backups/create-restic-repository-shortguide/index.md

```diff
@@ -41,31 +45,43 @@ the repository. Losing your password means that your data is
 irrecoverably lost.
 {{< /output >}}
 
-{{< caution >}}
+    {{< caution >}}
 Store this password securely and somewhere away from your Linode. Your backups will be inaccessible without it!
 {{< /caution >}}
 
 ### Store the access key and secret
 
 Your access key, secret key and password are required every time Restic communicates with your repository. To make it easier to work with your repository, create a shell script containing your credentials.
 
-To keep it secure, create this script within the root user's home directory, and run all your Restic scripts as the root user.
+1. To keep it secure, create this script within the root user's home directory, and run all your Restic scripts as the root user.
 
-    sudo nano /root/restic_params
+        sudo nano /root/restic_params
 
-{{< file "/root/restic_params" >}}export AWS_ACCESS_KEY_ID=your-key
+    {{< file "/root/restic_params" >}}
+export AWS_ACCESS_KEY_ID=your-key
 export AWS_SECRET_ACCESS_KEY=your-secret
 {{< /file >}}
 
+    {{< note >}}
 Whenever you want to use Restic, import this file or include it in your user's logon script:
 
     source /root/restic_params
+{{< /note >}}
+
+1. Create a password file to hold your Restic password:
 
-Create a password file to hold your Restic password:
+        sudo nano /root/restic_pw
 
-{{< file "/root/restic_pw" >}}YourPasswordGoesHere
+    {{< file "/root/restic_pw" >}}
+YourPasswordGoesHere
 {{< /file >}}
 
+    {{< note >}}
 You can pass your password filename to Restic using the "-p" flag:
 
-    restic -p /root/restic_pw ...
\ No newline at end of file
+    restic -p /root/restic_pw ...
+{{< /note >}}
+
+1. Run the `chmod` command to restrict read access for the new files to the root user:
+
+        sudo chmod 600 /root/restic_params /root/restic_pw
```

Comment:
The files that were created in this section were readable by groups and other users, which I thought was not a good idea for files that contain secrets. But, maybe that's not an issue for some reason I'm not thinking of?

This change had implications in the main guide, where sudo needed to be used in several places that it wasn't used before

Reply:
> The files that were created in this section were readable by groups and other users, which I thought was not a good idea for files that contain secrets.

I totally agree, my reasoning for not explicitly chmod’ing them was that they are created in /root - which is only accessible to the root user.

Reply:
ah, this was my mistake--I was forgetful that the file wouldn't be readable without the group/other execute bit set on `/root`. Since this is redundant, I'll push up a commit to revert the `chmod` step